### PR TITLE
Fix Rails 5 compatibility issues

### PR DESCRIPTION
**Pull request by Stankec**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,14 +3,14 @@ source "https://rubygems.org"
...
 
-rails_version = ENV["RAILS_VERSION"] || "4.1.4"
+rails_version = ENV["RAILS_VERSION"] || "5.0.0"
 
 gem "rails", "~> #{rails_version}"
 
 gem 'factory_girl_rails'
 gem 'sqlite3'
 gem 'rspec-rails'
-gem 'capybara', '~> 2.3.0'
```
Comment (DamirSvrtan) [inline Gemfile]: Hey @Stankec - is this bump due to compatibility issues with Rails 5?

File: spec/dummy/spec/javascripts/support/jasmine.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  - "helpers/**/*.{js.coffee,js,coffee}"
+
+# list of file expressions to include as specs into spec runner
+# relative path from spec_dir
+spec_files:
+  - "**/*[Ss]pec.{js.coffee,js,coffee}"
+
+# path to directory of temporary files
+# (spec runner and asset cache)
+# defaults to tmp/jasmine
```
Comment (DamirSvrtan) [inline spec/dummy/spec/javascripts/support/jasmine.yml]: what's going on here? Was this generated during the test run?

File: app/controllers/phrasing_phrases_controller.rb [hunk 3]
```diff
@@ -6,7 +6,7 @@ class PhrasingPhrasesController < Phrasing.parent_controller.constantize
 
   include PhrasingHelper
 
-  before_filter :authorize_editor
+  before_action :authorize_editor
```
Comment (DamirSvrtan) [inline app/controllers/phrasing_phrases_controller.rb]: Rails 5 finally doesn't support before_filters (switched completely to before_actions)?

Comment (DamirSvrtan) [inline app/controllers/phrasing_phrases_controller.rb]: If that's so, I'm okay with removing the Rails 3.2.0 support from travis. If this is just a style change, I would leave it as a before filter.

File: Gemfile (hunk 1 above)
Comment (Stankec) [inline Gemfile]: Yes. Older versions of Capybara use now-deprecated Rack helper method. As Rails 5 has a hard dependency on Rack 2 a version bump was required.

File: app/controllers/phrasing_phrases_controller.rb (hunk 3 above)
Comment (Stankec) [inline app/controllers/phrasing_phrases_controller.rb]: Rails 5 finally dropped support for `*_filter` methods. I will modify the `.travis.yml` file if it's ok to drop Rails 3.2.0 support.

File: spec/dummy/spec/javascripts/support/jasmine.yml (hunk 2 above)
Comment (Stankec) [inline spec/dummy/spec/javascripts/support/jasmine.yml]: Capybara uses Jasmin for some reason. This is an autogenerated file. I have to say that I'm not quite sure what it's used for. It regenerates each time you run the test suite so I guess it's important.

**Comment by Stankec** [comment]
yay

**Pull request merged**